tests/foreman/ui/test_host.py

```diff
@@ -2461,3 +2461,46 @@ def _cleanup():
             assert (
                 host.run(f'dnf remove {" ".join(packages)} -y').status == 0
             ), 'Could not remove installed packages in a finalizer!'
+
+
+@pytest.mark.tier2
+def test_all_hosts_bulk_hostgroup_assignment(target_sat, module_org, module_location, new_host_ui):
```

Comment:
Remove `new_host_ui` if not using anywhere
```suggestion
def test_all_hosts_bulk_hostgroup_assignment(target_sat, module_org, module_location):
```

Reply:
I think this fixture is used just to enable the new host UI on the SAT, so we need to keep it.